### PR TITLE
Update table-storage.md for Cosmos DB Table API

### PR DESCRIPTION
**Pull request by MarkMcGeeAtAquent**
## Summary

I made updates to document the Cosmos DB Table API support.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: docs/fsharp/using-fsharp-on-azure/table-storage.md
Comment (pkulikov) [inline docs/fsharp/using-fsharp-on-azure/table-storage.md]: Please, replace fwlink with [the actual link](https://docs.microsoft.com/en-us/dotnet/api/overview/azure/storage?view=azure-dotnet) as the issue #3391 really against fwlinks :).

**Comment by MarkMcGeeAtAquent** [comment]
Not yet ready for review--

File: docs/fsharp/using-fsharp-on-azure/table-storage.md
Comment (MarkMcGeeAtAquent) [reply docs/fsharp/using-fsharp-on-azure/table-storage.md]: Ah, OK, fixed.

File: docs/fsharp/using-fsharp-on-azure/table-storage.md
Comment (mairaw) [inline docs/fsharp/using-fsharp-on-azure/table-storage.md]: not related to your change, but do you mind updating this link to https://azure.microsoft.com/resources/samples/storage-table-dotnet-getting-started/?

File: docs/fsharp/using-fsharp-on-azure/table-storage.md
Comment (mairaw) [inline docs/fsharp/using-fsharp-on-azure/table-storage.md]: this link should go to https://docs.microsoft.com/azure/storage/common/storage-configure-connection-string instead

File: docs/fsharp/using-fsharp-on-azure/table-storage.md
Comment (mairaw) [inline docs/fsharp/using-fsharp-on-azure/table-storage.md]: you don't whether -> you don't know whether

File: docs/fsharp/using-fsharp-on-azure/table-storage.md
Comment (mairaw) [inline docs/fsharp/using-fsharp-on-azure/table-storage.md]: also not sure if these levels are correct. it seems they should also be H3?

File: docs/fsharp/using-fsharp-on-azure/table-storage.md
Comment (mairaw) [inline docs/fsharp/using-fsharp-on-azure/table-storage.md]: not related to your change, but can you fix line 49
 after `Microsoft.WindowsAzure.ConfigurationManager` it should be a backtick instead of a single quote?

**Review by mairaw — CHANGES_REQUESTED** [review]
some quick comments on things that need to get addressed but I'd also like to get @cartermp review before merging since this is for F#. Thanks @MarkMcGeeAtAquent for the updates!

File: docs/fsharp/using-fsharp-on-azure/table-storage.md
Comment (mairaw) [inline docs/fsharp/using-fsharp-on-azure/table-storage.md]: remove please

File: docs/fsharp/using-fsharp-on-azure/table-storage.md
Comment (mairaw) [inline docs/fsharp/using-fsharp-on-azure/table-storage.md]: nit: [use contractions](https://docs.microsoft.com/en-us/style-guide/word-choice/use-contractions)

If you are -> If you're

File: docs/fsharp/using-fsharp-on-azure/table-storage.md
Comment (mairaw) [inline docs/fsharp/using-fsharp-on-azure/table-storage.md]: you'll enter -> enter

like this -> like the following example

File: docs/fsharp/using-fsharp-on-azure/table-storage.md
Comment (mairaw) [inline docs/fsharp/using-fsharp-on-azure/table-storage.md]: Some existing issues pre-acrolinx:

line 82 : This will return -> This returns

line 84: will be stored -> are stored

line 92: Now we'll add our `Customer` to the table. To do so, you create a `TableOperation` that will execute on the table. ->
Now add `Customer` to the table. To do so, create a `TableOperation` that executes on the table.

File: docs/fsharp/using-fsharp-on-azure/table-storage.md
Comment (mairaw) [inline docs/fsharp/using-fsharp-on-azure/table-storage.md]: line 160 and 188: will fail -> fails

**Review by mairaw — COMMENTED** [review]
A few more comments to improve this topic

File: docs/fsharp/using-fsharp-on-azure/table-storage.md
Comment (MarkMcGeeAtAquent) [reply docs/fsharp/using-fsharp-on-azure/table-storage.md]: fixed

File: docs/fsharp/using-fsharp-on-azure/table-storage.md
Comment (MarkMcGeeAtAquent) [reply docs/fsharp/using-fsharp-on-azure/table-storage.md]: fixed

File: docs/fsharp/using-fsharp-on-azure/table-storage.md
Comment (MarkMcGeeAtAquent) [reply docs/fsharp/using-fsharp-on-azure/table-storage.md]: fixed

File: docs/fsharp/using-fsharp-on-azure/table-storage.md
Comment (MarkMcGeeAtAquent) [reply docs/fsharp/using-fsharp-on-azure/table-storage.md]: fixed

File: docs/fsharp/using-fsharp-on-azure/table-storage.md
Comment (MarkMcGeeAtAquent) [reply docs/fsharp/using-fsharp-on-azure/table-storage.md]: fixed

**Review by cartermp — APPROVED** [review]

**Comment by MarkMcGeeAtAquent** [comment]
@mairaw , I've made the fixes you requested, but I'm not familiar with the PR workflow for this repo. Do you need to sign off on the PR, or do I, or ? 
Thanks!

**Comment by cartermp** [comment]
We'll just need @mairaw to approve and this will be good to merge

**Comment by mimig1** [comment]
The Azure Cosmos DB changes look good to me.

**Comment by mairaw** [comment]
There was an extra line added to the sample code that I think made the content samples break. Removed it now to test it.

Staging URL is https://review.docs.microsoft.com/en-us/dotnet/fsharp/using-fsharp-on-azure/table-storage?branch=pr-en-us-4745 

For example:
image

**Comment by mairaw** [comment]
Build seemed stuck.

**Comment by MarkMcGeeAtAquent** [comment]
Some of the snippets are still not displaying (e.g., Delete a table). I don't understand the cause of the bug, but as an alternative we can hardcode the snippets into the article.

**Comment by MarkMcGeeAtAquent** [comment]
I think I found the problems

**Comment by MarkMcGeeAtAquent** [comment]
@mairaw, all F# code is displaying correctly, ready to publish

**Comment by mairaw** [comment]
Not really. This one was incomplete:
image

**Review by mairaw — APPROVED** [review]
PR should be good to go after my last changes. Waiting for build to finish in order to merge it.

**Pull request merged**
